### DISCUSSION TITLE
[Feature]enable automatic library scanning for external library

### DISCUSSION THREAD

**Discussion by JackMai123** [immich-app/immich#6322] [Feature Request]
### The feature

In the admin, manage library settings, have the external library scanned for new files when enabled.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by aisbergde** [comment]
automatic scheduled rescan is enabled by default, as I know. You can even change the frequency for the implemented cron job.

Or do you mean something different?

**Reply by JackMai123** [reply]
That is what I thought too, but the scheduled scan doesn't seem to pickup new files added to the external source. Maybe this is just a bug or am I missing something...

**Reply by aisbergde** [reply]
are you using v1.92? There was a bug on the previous version and it was not running.

**Reply by JackMai123** [reply]
I've upgraded to the latest version and it is working!!! Thanks you!
